rootfs_linux.go

```diff
@@ -62,6 +69,21 @@ func setupRootfs(config *configs.Config, console *linuxConsole) (err error) {
 	return nil
 }
 
+func mountCmd(cmd string) error {
+	if cmd != "" {
+		args := strings.Split(cmd, " ")
+		mCmd := exec.Command(args[0], args[1:]...)
```

Comment:
# github.com/docker/libcontainer
vendor/src/github.com/docker/libcontainer/rootfs_linux.go:74: not enough arguments in call to exec.Command